substrait-csharp.sln

```diff
@@ -1,20 +1,20 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 25.0.1700.0
+# Visual Studio Version 17
+VisualStudioVersion = 17.3.33027.108
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "substrait-core", "core\substrait-core.csproj", "{D0621DA5-91E4-4273-A076-CF7B15EC0139}"

```

Comment:
Should you delete 'core/substrait-core.csproj'? That seems to be missing in this PR.

Reply:
Good catch. The file is now removed.